### PR TITLE
Fix Example Projects and run them via CI

### LINKED ISSUE #970: Testable Swift Example Issue
Hey,

I ran one of example(Testable Swift) from downloaded KIF repo and found below error: 

**Assertions: The step timed out after 10.00 seconds: Failed to find accessibility element with the label "Red"
File: SimpleObjCTest.m:20**

I found that this issue is related to accessibility and tried to debug but couldn't find anything. 
I am using XCode 8.3.2 and device(or simulator) running iOS 10.3.

Thanks in advance.

### PR DESCRIPTION
**Pull request by justinseanmartin**
Fixes #970

Also, updated it to use the examples to use the v3 API that @RoyalPineapple introduced.

CC: @imsrc21 @phatmann

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 language: objective-c
 matrix:
   include:
-    - osx_image: xcode6.4
-      env: 'SIMULATOR="name=iPad Air,OS=8.1"'
+    - osx_image: xcode8.3
```
Comment (phatmann) [inline .travis.yml]: Good idea.

File: Classes/KIFUITestActor.m [hunk 2]
```diff
@@ -387,24 +387,32 @@ - (void)enterTextIntoCurrentFirstResponder:(NSString *)text
...
+    UIView *fallback = fallbackView;
+    if (!fallback) {
+        UIResponder *firstResponder = [[[UIApplication sharedApplication] keyWindow] firstResponder];
+
+        if ([firstResponder isKindOfClass:[UIView class]]) {
+            fallback = (UIView *)firstResponder;
+        }
+    }
+
+    id<UITextInputTraits> textInputField = nil;
```
Comment (phatmann) [inline Classes/KIFUITestActor.m]: Finally a fix for this issue.

**Comment by phatmann** [comment]
I will run Travis again now.

File: Classes/KIFUITestActor.m (hunk 2 above)
Comment (justinseanmartin) [inline Classes/KIFUITestActor.m]: Yea, saw it appearing on the integration tests on iOS10 and thought I'd better fix it. I could just change the app to disable autocorrection, but I figured if I'm already in here, I might as well fix it for all text controls.

**Comment by justinseanmartin** [comment]
I need to take a look at my changes to the CI script, for some reason `Acceptance Tests` is failing on Travis.

File: Classes/KIFUITestActor.m (hunk 2 above)
Comment (justinseanmartin) [inline Classes/KIFUITestActor.m]: Unfortunately doesn't seem to work: https://travis-ci.org/kif-framework/KIF/jobs/229223229

```
    ✗ testEnteringEmojiCharactersIntoViewWithAccessibilityLabel, The step timed out after 10.00 seconds: Failed to get text " 😓He😤ll👿o" in field; instead, it was " 😓He😤lol👿o"
```

**Comment by justinseanmartin** [comment]
Fixed the script (was boneheaded mistake), but still the iOS10 job is broken for the `testEnteringEmojiCharactersIntoViewWithAccessibilityLabel` test. I'll need to look more closely at this.

**Comment by justinseanmartin** [comment]
Don't merge this, but I'm seeing if I just make the control itself on initialization to be `UITextAutocorrectionTypeNo` if that will even allow the test to pass. I can't reproduce the failure locally, but I'm still on Xcode 8.2.1 instead of 8.3.2. Don't know if that's it or not though, seems like it shouldn't be.

**Comment by justinseanmartin** [comment]
Apparently the changes in 0604fe2 fix the test, but the changes in ed296f1 do not. Very strange... Will do some more digging.

**Comment by justinseanmartin** [comment]
I'm going to split this out into two PRs. One for the original issue in #972 and a separate one to add an iOS10 Xcode 8.3 build configuration (which has to include fixing that typing test).

**Comment by justinseanmartin** [comment]
Ok @phatmann & @RoyalPineapple, please take a look. This now just updates the sample projects to all pass, adds a CI configuration that runs them, and converts them to use the KIFUIViewTestActor's `viewTester` syntax.

**Comment by mikelupo** [comment]
@justinseanmartin thank you for this update. Looks like Travis is happy. I like the change. +1 from me.

**Pull request merged**